content/blog/2020-01-ggplot2-3-3-0/index.Rmarkdown

```diff
@@ -0,0 +1,231 @@
+---
+title: "ggplot2 3.3.0"
+author: "Thomas Lin Pedersen"
+date: '2020-01-07'
+description: |
+  The next iteration of ggplot2 has just been released on CRAN, packed with new 
+  features and bug fixes. Read all about what's new here.
+photo:
+  author: Steve Harvey
+  url: https://unsplash.com/photos/xWiXi6wRLGo
+slug: ggplot2-3-3-0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're so happy to announce the release of ggplot2 3.3.0 on CRAN. ggplot2 is a system for declaratively creating graphics, based on *The Grammar of Graphics*. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
```

Comment:
```suggestion
We're so happy to announce the release of ggplot2 3.3.0 on CRAN. [ggplot2](https://ggplot2.tidyverse.org/) is a system for declaratively creating graphics, based on *The Grammar of Graphics*. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
```

Reply:
I usually just link to the pkgdown page when I do the description.

Reply:
I also add a sentence here (at the end of the first paragraph) linking to the full Changelog or News "For a full list of changes in this release..."). Once the pkgdown site has 3.3.0 in the changelog, I think that's the best place. Or you can literally reference [NEWS.md](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md).